config/prod.webpack.config.js

```diff
@@ -9,7 +9,11 @@ const { config: webpackConfig, plugins } = config({
 
 plugins.push(
     require('@redhat-cloud-services/frontend-components-config/federated-modules')({
-        root: resolve(__dirname, '../')
+        root: resolve(__dirname, '../'),
+        exposes: {
+            //ZeroStateBanner
```

Comment:
Also, wouldn't we export `ZeroState` instead of `ZeroStateBanner`?

Reply:
This is the schema adopted in other repos like inventory. 

I exported ZeroStateBanner because not all applications will use this entire zero state, just parts of it.